src/workerPools.js

```diff
@@ -3,10 +3,23 @@ import WorkerPool from './WorkerPool';
 
 const workerPools = Object.create(null);
 
+function calculateNumberOfWorkers() {
+  const cpus = os.cpus();
```

Comment:
In some os you can get `undefined` on `os.cpus()` use this `const cpus = os.cpus() || { length: 1 };`. Maybe we can simplify check, look what we use in terser plugin https://github.com/webpack-contrib/terser-webpack-plugin/blob/master/src/TaskRunner.js#L20